modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultCodegen.java

```diff
@@ -130,10 +130,10 @@ public void processOpts() {
     @SuppressWarnings({ "static-method", "unchecked" })
     public Map<String, Object> postProcessAllModels(Map<String, Object> objs) {
         if (supportsInheritance) {
-            // Index all CodegenModels by name.
+            // Index all CodegenModels by model name.
             Map<String, CodegenModel> allModels = new HashMap<String, CodegenModel>();
             for (Entry<String, Object> entry : objs.entrySet()) {
-                String modelName = entry.getKey();
+                String modelName = toModelName(entry.getKey());
```

Comment:
Typescript and PHP also set supportsInheritance to `true`: https://github.com/swagger-api/swagger-codegen/search?utf8=%E2%9C%93&q=supportsInheritance

This is not to say your change will break anything. I'm just trying to evaluate the impact of the change.

What about merging it by coming Monday instead so that I can have a bit more time to review?